src/core/public/chrome/ui/header/header_breadcrumbs.tsx

```diff
@@ -57,7 +59,15 @@ export function HeaderBreadcrumbs({ appTitle$, breadcrumbs$ }: Props) {
       i === 0 && 'first',
       i === breadcrumbs.length - 1 && 'last'
```

Comment:
Are these still needed?

Reply:
yes! this is from data-test-subj! this part we are not touching!